.github/workflows/zap-scan-test.yml

```
@@ -1,44 +1,39 @@
-
 name: ZAP-Scan-Test
 
-# Controls when the action will run.
 on:
-  pull_request:
-    branches:
-      - main
+   pull_request:
+     branches:
+       - main
+  # Weekly scheduled scan of the deployed test environment
```

actionlint (1.7.12)

[error] could not parse as YAML: did not find expected key

(syntax-check)